backend/capellacollab/feedback/metrics.py

```diff
@@ -29,5 +29,23 @@ def collect(self) -> t.Iterable[prometheus_client.core.Metric]:
         yield metric
 
 
+class FeedbackAnonymityCollector(prometheus_registry.Collector):
```

Comment:
I'd handle it slightly different. Instead of a separate metric, I'd just embed it into the existing metric as label. The result would be: 

```
feedback_count{rating="bad",anonymous="1"} 0.0
feedback_count{rating="bad",anonymous="0"} 0.0
feedback_count{rating="okay",anonymous="1"} 0.0
feedback_count{rating="okay",anonymous="0"} 0.0
feedback_count{rating="good",anonymous="1"} 0.0
feedback_count{rating="good",anonymous="0"} 0.0
```